readme.md

```diff
@@ -12,3 +12,25 @@ A fun side project by some lovely folks at [Tighten Co.](http://tighten.co/).
 
 * [Api documentation](doc/api.md)
 * Chat about it on freenode IRC channel `#symposium`
+
+### Requirements
+
+* PHP >= 5.5.9
+* PHP [mcrypt extension](http://php.net/manual/en/book.mcrypt.php)
+* A [supported relational database](http://laravel.com/docs/5.1/database#introduction) and corresponding PHP extension
+* [Composer](https://getcomposer.org/download/)
+
+### Installation
+
+1. [Fork this repository](https://help.github.com/articles/fork-a-repo/)
+2. [Install dependencies](https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies)
+3. [Run database migrations](http://laravel.com/docs/5.1/migrations#running-migrations) against the desired database using the appropriate database connection name from [configuration](https://github.com/tightenco/symposium/blob/master/config/database.php)
+
+    ```
+    php artisan migrate --database=sqlite
+    ```
+4. Configure a web server to use the `public` directory as the document root and the appropriate database connection name as the `DB_CONNECTION` environmental variable.
```

Comment:
We'll need to update this; Laravel actually use `phpdotenv`, so if we're relying on a `DB_CONNECTION` value, it should come from there.

Additionally, any installation instructions should include copying `.env.example` to `.env` and setting information there.